arlo-client/src/components/Audit/StatusBox.tsx

```diff
@@ -0,0 +1,198 @@
+import React from 'react'
+import { useParams } from 'react-router-dom'
+import styled from 'styled-components'
+import { Callout, H4 } from '@blueprintjs/core'
+import { toast } from 'react-toastify'
+import useRoundsAuditAdmin, { IRound } from './useRoundsAuditAdmin'
+import useJurisdictions, {
+  IJurisdiction,
+  FileProcessingStatus,
+  JurisdictionRoundStatus,
+} from './useJurisdictions'
+import FormButton from '../Atoms/Form/FormButton'
+import { api } from '../utilities'
+import { Inner } from '../Atoms/Wrapper'
+import { IAuditSettings, IContest } from '../../types'
+import useAuditSettings from './useAuditSettings'
+import useContests from './useContests'
+
+const Wrapper = styled(Callout)`
+  display: flex;
+  padding: 30px 0;
+  .text {
+    flex-grow: 1;
+    p {
+      margin-bottom: 0;
+    }
+  }
+`
+enum Button {
+  START_NEXT_ROUND,
+  DOWNLOAD_REPORT,
+}
+
+const createRound = async (electionId: string, roundNum: number) => {
+  try {
+    api(`/election/${electionId}/round`, {
+      method: 'POST',
+      body: JSON.stringify({
+        roundNum,
+      }),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    })
+  } catch (err) {
+    toast.error(err.message)
+  }
+}
+
+const downloadAuditReport = (electionId: string) => {
+  window.open(`/election/${electionId}/audit/report`)
+}
+
+export const isSetupComplete = (
+  jurisdictions: IJurisdiction[],
+  contests: IContest[],
+  auditSettings: IAuditSettings
+): boolean =>
+  jurisdictions.length > 0 &&
+  contests.some(c => c.isTargeted) &&
+  Object.entries(auditSettings).every(([, v]) => v !== null)
```

Comment:
Not blocking, just curious, since I haven't used `Object.entries` before. But isn't this equivalent to `Object.values(auditSettings).every(v => v !== null)`?

Reply:
Oh dang right as I clicked the merge button! Yes, you're right, I'll swap that in